server/tserver/src/main/java/org/apache/accumulo/tserver/metrics/TabletServerMetrics.java

```diff
@@ -36,6 +38,13 @@ public TabletServerMetrics(TabletServer tserver) {
 
   @Override
   public void registerMetrics(MeterRegistry registry) {
+    FunctionCounter
+        .builder(METRICS_COMPACTOR_ENTRIES_READ, null, o -> FileCompactor.getTotalEntriesRead())
+        .description("Number of entries read").tag("type", "tserver").register(registry);
```

Comment:
Could add a bit more to the description.

```suggestion
        .description("Number of entries read by all compactions that have run in the process").tag("type", "tserver").register(registry);
```